### PR TITLE
Noting all sections must be completed for secure comms.

### PR DESCRIPTION
**Pull request by asachs01**
## Description
Per https://github.com/sensu/sensu-docs/pull/1079#issuecomment-454089009, there is additional clarity that needed to be provided to avoid causing agent comms to cease. This PR updates the doc with more explicit language around securing agent/server comms.

### CODE REVIEW

File: content/sensu-go/5.0/guides/securing-sensu.md [hunk 1]
```diff
@@ -67,7 +69,9 @@ In the example above, we provide the path to the cert, key and CA file. After re
 
 ## Securing Sensu agent to server communication
 
-The Sensu agent uses WebSockets for communication between the agent and the server. By default, an agent uses the insecure `ws://` transport. Let's look at the example from `/etc/sensu/agent.yml`:
+We'll now discuss securing agent to server communication. Please note: by changing the agent configuration to communicate via WebSocket Secure protocol, the agent will no longer communicate over a plaintext connection. If the server is not secured as in the [section above](#securing-the-api-and-the-dashboard), communication between the agent and server will not function.
```
Comment (apaskulin) [inline content/sensu-go/5.0/guides/securing-sensu.md]:
```suggestion
We'll now discuss securing agent to server communication. Please note: by changing the agent configuration to communicate via WebSocket Secure protocol, the agent will no longer communicate over a plaintext connection. If the server is not secured as described in the [section above](#securing-the-api-and-the-dashboard), communication between the agent and server will not function.
```

**Review by apaskulin — COMMENTED** [review]
This looks great. Thanks for making this update. Can you also update this sentence "While enabling secure agent-to-server communication involves the change of one line"?

**Comment by asachs01** [comment]
Closes #1018

**Review by apaskulin — APPROVED** [review]
Looks great! +1 to copy to 5.1

**Pull request merged**